tests/unit/src/test/scala/tests/CompletionLspSuite.scala

```diff
@@ -294,4 +294,53 @@ class CompletionLspSuite extends BaseCompletionLspSuite("completion") {
       )
     } yield ()
   }
+
+  test("symbolic-name") {
+    cleanWorkspace()
+    for {
+      _ <- initialize(
+        """/metals.json
+          |{
+          |  "a": {}
+          |}
+          |/a/src/main/scala/a/A.scala
+          |package a
+          |
+          |abstract class Base {
+          |  /**
+          |    * Some documentation
+          |    */
+          |  type !![A, B] = A with B
+          |}
+          |object Main extends Base {
+          |  // @@
+          |}
+          |""".stripMargin
+      )
+      _ <- assertCompletionItemResolve(
+        "val x = List(1).:::@@",
+        expectedLabel = ":::[B >: Int](prefix: List[B]): List[B]",
+        expectedDoc = Some(
+          """|Adds the elements of a given list in front of this list.
+             |
+             |Example:
+             |
+             |```
+             |List(1, 2) ::: List(3, 4) = List(3, 4).:::(List(1, 2)) = List(1, 2, 3, 4)
+             |```
+             |**Parameters**
+             |- `prefix`: The list elements to prepend.
+             |
+             |**Returns:** a list resulting from the concatenation of the given
+             |   list `prefix` and this list.
+             |""".stripMargin
+        ),
+      )
+      _ <- assertCompletionItemResolve(
+        "val x: !!@@",
+        expectedLabel = "!!",
```

Comment:
Question (sorry if doesn't make sense, just caught my attention), for the `:::` from Scala's `List` above we get label with much more info (generic type signature, return type, etc.), but here for the `!!` we only see the symbol name. Is it expected? :slightly_smiling_face:

Reply:
This is a valid question. Basically, for type completions, this information is not a part of label, but is provided in `detail` (in vscode its in the same place as doc)